lib/constants.go

```diff
@@ -1,30 +1,44 @@
 package libunlynx
 
 import (
+	"go.dedis.ch/onet/v3/log"
+	"os"
 	"sync"
+	"time"
 
 	"go.dedis.ch/onet/v3/simul/monitor"
 )
 
+func init() {
+	timeout, err := time.ParseDuration(os.Getenv("MEDCO_TIMEOUT"))
```

Comment:
Is it wise here to mention MedCo? Shouldn't Unlynx be an independent piece of software?

Reply:
I can rename this environnent variable :) Any suggestions? Maybe COM_TIMEOUT (for communication timeout)?

Reply:
CONN_TIMEOUT ? :)

Reply:
done yours is a better name!